pages/mesosphere/dcos/services/beta-jenkins/4.0.0-2.204.2-beta/quickstart/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: layout.pug
+navigationTitle:  Jenkins for DC/OS Quickstart
+title: Jenkins for DC/OS Quickstart
+menuWeight: 10
+excerpt: Jenkins for DC/OS can be installed using either the web interface or the DC/OS CLI.
+beta: true
+featureMaturity:
+enterprise: false
+---
+# About installing Jenkins for DC/OS
+
+As a package available in the Universe, Jenkins for DC/OS can be installed using either the web interface or the DC/OS CLI. With its sensible defaults, you can get up and running very quickly.
+
+**Important:** The default installation will use a `/tmp` directory on the local host to store configuration and build data. This configuration will not scale to accommodate multiple Jenkins masters. In addition, it will result in the loss of data when the agent goes down. Before going into production, you must perform a [custom install](../custom-install/index.md) and set up either a shared file system (recommended) or pin to a single agent.
+
+# Prerequisites
+
+DC/OS 2.1 or later
```

Comment:
Oh. Didn't realize the minimum version required is 2.1!?

Reply:
This is up for change, since we have Windows agents only available in 2.1 I made it the minimum version, the previous one was 1.10, should I leave it at that?